Doc/library/profile.rst

```diff
@@ -54,7 +54,7 @@ This section is provided for users that "don't want to read the manual." It
 provides a very brief overview, and allows a user to rapidly perform profiling
 on an existing application.
 
-To profile a function that takes a single argument, you can do::
+To profile a statement with the namespace of ``__main__``, you can do::
```

Comment:
```suggestion
To profile a statement in the current namespace, you can do::
```

Reply:
Actually, you can't :). `run` will literally use the `__main__` namespace, not the local one.

Reply:
Indeed.
```suggestion
To profile a statement with the namespace of :mod:`__main__`, you can do::
```
Or "in the top-level code environment".

Reply:
I don't think "top-level code enviroment" is clearer than "__main__", but using the module might help.